main.cpp

```diff
@@ -183,105 +302,264 @@ void Vinyl::moveToneArm(bool is12Inch)
 {
     float distanceToMoveArm;
     distanceToMoveArm = is12Inch ? 6.0f : 3.5f; 
+    std::cout << "Move tone arm " << distanceToMoveArm << "inches" << std::endl;
+}
+
+double Vinyl::bringJoy(int setNumberOfGoodTracks, double setListenerMood)
+{
+    double quality = double(setNumberOfGoodTracks) * setListenerMood;
```

Comment:
no casting yet.  it hasn't been introduced.
change the type of the function argument to eliminate the warning for now
- [ ] revise